### PR TITLE
{bio}[foss/2019b] PheWeb v1.1.20 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/4cdf1365ca990c9202f8ceedd25da9cc for a full test report.

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Pysam', '0.15.3'),
+    ('openpyxl', '3.0.3', versionsuffix, ('GCCcore', '8.3.0')),
+    ('Flask', '1.1.2', versionsuffix, ('GCCcore', '8.3.0')),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:19]: is specifying the subtoolchain necessary?

Comment (verdurin) [reply easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:19]: @migueldiascosta certainly open to better suggestions here - I used the first formulation I found that worked.

Comment (migueldiascosta) [reply easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:19]:
```
    ('openpyxl', '3.0.3', versionsuffix),
    ('Flask', '1.1.2', versionsuffix),
```

works for me... but maybe it depends on some easybuild toolchain-related configuration setting you are using?

Comment (branfosj) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:19]:
```suggestion
    ('openpyxl', '3.0.3', versionsuffix),
    ('Flask', '1.1.2', versionsuffix),
```

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }),
+    ('gunicorn', '20.0.4', {
+        'checksums': ['1904bb2b8a43658807108d59c3f3d56c2b6121a701161de0ddf9ad140073c626'],
+    }),
+    ('boltons', '19.1.0', {
+        'checksums': ['c32b2d121331a9bc7c220050d4273f3aa359b7569cb4794188e71524603113dc'],
+    }),
+    ('blist', '1.3.6', {
+        'checksums': ['3a12c450b001bdf895b30ae818d4d6d3f1552096b8c995f0fe0c74bef04d1fc3'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:61]:
```suggestion
```

In Python/3.7.4

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    ('Pysam', '0.15.3'),
+    ('openpyxl', '3.0.3', versionsuffix, ('GCCcore', '8.3.0')),
+    ('Flask', '1.1.2', versionsuffix, ('GCCcore', '8.3.0')),
+    ('tqdm', '4.41.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:26]:
```suggestion
```

Remove default.

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb [hunk 4]
```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'PheWeb'
+version = '1.1.20'
+versionsuffix = '-Python-%(pyver)s'
+
+
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:7]:
```suggestion
```

Remove extra blank line.

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+homepage = 'https://github.com/statgen/pheweb'
+description = """A tool for building PheWAS websites from association files"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:13]:
```suggestion
```

Remove extra blank line.

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+    }),
+    ('sortedcontainers', '2.2.2', {
+        'checksums': ['4e73a757831fc3ca4de2859c422564239a31d8213d09a2a666e375807034d2ba'],
+    }),
+    ('intervaltree', '3.0.2', {
+        'checksums': ['cb4f61c81dcb4fea6c09903f3599015a83c9bdad1f0bbd232495e6681e19e273'],
+    }),
+    ('requests', '2.24.0', {
+        'checksums': ['b3559a131db72c33ee969480840fff4bb6dd111de7dd27c8ee1f820f4f00231b'],
+    }),
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb:52]:
```suggestion
```

`intervaltree` is in Python/3.7.4 and sufficient `sortedcontainers` (2.1.0) and `requests` (2.22.0) are also there

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb (hunk 1 above)
Comment (branfosj) [reply easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb]: done in 6d6131f

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/c37e264f7e2acc99c5a6356e49ab4a3e for a full test report.

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @verdurin!

**Pull request merged**